synapse/rest/client/v2_alpha/account.py

```diff
@@ -53,21 +53,11 @@ def __init__(self, hs):
         self.identity_handler = hs.get_handlers().identity_handler
 
         if self.config.threepid_behaviour_email == ThreepidBehaviour.LOCAL:
-            template_html, template_text = load_jinja2_templates(
-                self.config.email_template_dir,
-                [
-                    self.config.email_password_reset_template_html,
-                    self.config.email_password_reset_template_text,
-                ],
-                apply_format_ts_filter=True,
-                apply_mxc_to_http_filter=True,
-                public_baseurl=self.config.public_baseurl,
-            )
             self.mailer = Mailer(
                 hs=self.hs,
                 app_name=self.config.email_app_name,
-                template_html=template_html,
-                template_text=template_text,
+                template_html=hs.config.email_password_reset_template_html,
+                template_text=hs.config.email_password_reset_template_text,
```

Comment:
I think we should be consistent with the line above this:

```suggestion
                template_html=self.config.email_password_reset_template_html,
                template_text=self.config.email_password_reset_template_text,
```